tldw_Server_API/app/core/Embeddings/dlq_crypto.py

```
@@ -10,14 +10,38 @@
+def _derive_key_from_passphrase(passphrase: str, salt: bytes) -> Tuple[bytes, str]:
+    import hashlib
+    try:
+        key = hashlib.scrypt(
+            passphrase.encode("utf-8"),
+            salt=salt,
+            n=_SCRYPT_PARAMS["n"],
+            r=_SCRYPT_PARAMS["r"],
+            p=_SCRYPT_PARAMS["p"],
+            dklen=_SCRYPT_PARAMS["dklen"],
+        )
+        return key, "scrypt"
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)